### PR TITLE
UI for group by

### PR DESCRIPTION
**Pull request by krokhmaliuk-virtru**

### CODE REVIEW

File: src/components/EditValueList/EditValueList.tsx [hunk 1]
```diff
@@ -1,37 +1,93 @@
...
+                        <Button
+                          style={{ left: '-1px' }}
+                          icon={<DeleteOutlined />}
+                          onClick={() => onRemoveHandle(remove, field.name)}
+                        />
+                     </div>
+                  ))}
+                  <Form.Item>
+                      <Button onClick={() => add('')}>
+                          Add field
```
Comment (sievdokymov-virtru) [inline src/components/EditValueList/EditValueList.tsx:71]: 'Add Value' button name would be clearer

Comment (krokhmaliuk-virtru) [reply src/components/EditValueList/EditValueList.tsx:81]: updated

File: src/components/EditValueList/EditValueList.test.tsx [hunk 2]
```diff
@@ -1,20 +1,70 @@
...
+        setWindowMock();
+        // @ts-ignore
+        const x = render(<EditValueList {...props} />);
+
+        expect(x.findByTestId("edit-value-input-field-0")).toBeTruthy();
+        expect(x.findByTestId("edit-value-input-field-1")).toBeTruthy();
+        expect(x.findByTestId("edit-value-input-field-2")).toBeTruthy();
+    });
+
+    it("should rendered with three inputs", () => {
```
Comment (sievdokymov-virtru) [inline src/components/EditValueList/EditValueList.test.tsx:57]: @krokhmaliuk-virtru with "no" inputs?

Comment (krokhmaliuk-virtru) [reply src/components/EditValueList/EditValueList.test.tsx]: fixed

File: src/components/EditValueList/EditValueList.tsx [hunk 3]
```diff
@@ -1,37 +1,93 @@
...
+    setSelectOptions(form.getFieldsValue().values.map(toOptions));
+  };
+
+  const onRemoveHandle = (remove: (index: number) => void, index: number) => {
+    remove(index);
+    onEdit(form.getFieldsValue().values);
+    setSelectOptions(form.getFieldsValue().values.map(toOptions));
+  };
+
+  const onChangeHandle = () => {
```
[the remark this answers is not in the document]
Comment (krokhmaliuk-virtru) [reply src/components/EditValueList/EditValueList.tsx]: outdated

File: src/components/EditValueList/EditValueList.tsx (hunk 1 above)
Comment (krokhmaliuk-virtru) [reply src/components/EditValueList/EditValueList.tsx]: decided replace to "Add new value"

**Review by sievdokymov-virtru — APPROVED** [review]

**Pull request merged**